cmd/axelard/cmd/root.go

```diff
@@ -88,6 +93,20 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 	return rootCmd, encodingConfig
 }
 
+func extendSeeds(cmd *cobra.Command, err error) error {
+	serverCtx := server.GetServerContextFromCmd(cmd)
+	seeds, err := config.ReadSeeds(serverCtx.Viper)
+	if err != nil {
+		return err
+	}
+
+	serverCtx.Config = config.MergeSeeds(serverCtx.Config, seeds)
+	if err := server.SetCmdServerContext(cmd, serverCtx); err != nil {
+		return err
+	}
+	return nil
```

Comment:
How about a info-level log messge here?  eg "Read 16 seeds from seeds.toml, 18 distinct seeds in total"